pkg/virt-operator/webhooks/webhook.go

```diff
@@ -90,3 +90,82 @@ func (k *KubeVirtDeletionAdmitter) Admit(review *v1beta1.AdmissionReview) *v1bet
 
 	return validating_webhooks.NewPassingAdmissionResponse()
 }
+
+func NewKubeVirtCreationAdmitter(client kubecli.KubevirtClient) *KubeVirtCreationAdmitter {
+	return &KubeVirtCreationAdmitter{
+		client: client,
+	}
+}
+
+type KubeVirtCreationAdmitter struct {
+	client kubecli.KubevirtClient
+}
+
+func (k *KubeVirtCreationAdmitter) Admit(review *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
+	var kv *v1.KubeVirt
+	var err error
+
+	kv, err = k.client.KubeVirt(review.Request.Namespace).Get(review.Request.Name, &metav1.GetOptions{})
```

Comment:
Kubevirt object should be in review.

Reply:
Sorry, I don't think I understood what you meant here

Reply:
review -> `v1beta1.AdmissionReview`
You don't need to get Kubevirt object, just marshal it from `review.Request.Object`